components/user/UserLinks.js

```diff
@@ -3,8 +3,10 @@ import Alert from "@components/Alert";
 
 export default function UserLinks({ BASE_URL, data }) {
   const defaultBucket = "Others";
-  data.links = data.links.map((link, i) => ({ id: i, ...link }));
-  const buckets = data.links.reduce((acc, obj) => {
+  // Solved bug 2.1 shown in the image
```

Comment:
what is `bug 2.1`?

Reply:
Bug 2.1 and 2.2 refers to the following screenshots screenshot, the second and third screenshots from the PR respectively:

<img width="1512" alt="Screenshot 2023-04-22 at 15 50 50" src="https://user-images.githubusercontent.com/61611599/234575448-ebd8c102-bf60-4429-9f10-6014381bffbf.png">
This is 2.1

<br/><br/>
<img width="1512" alt="Screenshot 2023-04-22 at 15 51 42" src="https://user-images.githubusercontent.com/61611599/234575498-f5de3f5c-8309-4e7d-bfec-9cbb17936c80.png">
This is 2.2

Reply:
Removing them

Reply:
Removed all those comments, can you check on your side?